packages/client/hmi-client/src/components/workflow/ops/interventions/tera-interventions-policy-card.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<div class="policy-card" :class="{ 'card-selected': selected }">
+		<div>
+			<span class="flex align-items-center"
+				><h6>{{ interventionPolicy.name }}</h6>
+				<Button
+					class="ml-auto"
+					text
+					icon="pi pi-ellipsis-v"
+					@click.stop="toggleContextMenu"
+				></Button
+			></span>
+			<ContextMenu ref="contextMenu" :model="contextMenuItems"></ContextMenu>
```

Comment:
- I don't think contextMenuItems need to be a `ref`
- I might suggest we use more explicit naming than `use`, as it will be a bit annoying to if we are searching for things.

Reply:
I've removed the ref
`use`->`use-intervention`